### ISSUE TITLE
The reproducing results.

### ISSUE THREAD

**Issue by SUNJIMENG** [Baekpica/xmcgan_torch#1]
Excuse me, have you ever trained your reproducing method? Did your pyTorch implementation of xmcgan reach a similar quantitative results as the paper, please?

**Comment by Baekpica** [comment]
No, these codes are not working properly. I stopped this implementation because I couldn't find the problematic part. I'm sorry I couldn't help you.

**Comment by YIRuriZhongtian** [comment]
Hi, I am trying to reproduce this paper recently, your code help me a lot. Can you tell me some details about your problems? Thanks!
